src/ReinforcementLearningCore/src/policies/agents/agent.jl

```diff
@@ -152,6 +152,14 @@ function get_dummy_action(action_space::AbstractVector)
     return action_space[1]
 end
 
+function get_dummy_action(action_space::Interval{:open,:closed})
+    return action_space.right
+end
+
+function get_dummy_action(action_space::Union{Interval{:closed,:open},Interval{:closed,:closed}})
+    return action_space.left
+end
+
```

Comment:
This will introduce an extra dependency of `IntervalSets` to `RLCore`. I'm not sure if it is worth adding it here. But if you insist, you'll need to update the https://github.com/JuliaReinforcementLearning/ReinforcementLearning.jl/blob/master/src/ReinforcementLearningCore/Project.toml (both the `deps` and `compat` sections)

Reply:
Not really worth it ;)